app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala

```diff
@@ -83,4 +91,5 @@ object AuditPayloadType {
   val CaseReferred = "caseReferred"
   val CaseReleased = "caseReleased"
   val CaseCompleted = "caseCompleted"
+  val RulingCancelled = "rulingCancelled"
```

Comment:
this needs to be reviewed - are there other actions for cancelling cases? or is your action the only cancel action?

Reply:
I don't think so but I will check with Joe.